packages/sharing-smart-contract/hardhat.config.cjs

```diff
@@ -40,6 +40,15 @@ module.exports = {
       url: 'https://bellecour.iex.ec',
       accounts: WALLET_PRIVATE_KEY ? [WALLET_PRIVATE_KEY] : [],
     },
+    // poco-chain native config
+    'dev-native': {
+      chainId: 65535,
```

Comment:
where does this chainId come from ?

Reply:
internal poco-chain repo => https://github.com/iExecBlockchainComputing/poco-chain